module/gurps.js

```diff
@@ -1927,6 +1929,8 @@ if (!globalThis.GURPS) {
     // @ts-ignore
     CONFIG.Actor.documentClass = GurpsActor
     CONFIG.Item.documentClass = GurpsItem
+    CONFIG.Combat.documentClass = CombatGURPS
+    CONFIG.Combatant.documentClass = CombatantGURPS
```

Comment:
If `combat` was a module, this could be moved into the `init` method.

Reply:
moved to its own module